build/signedbuild.yml

```diff
@@ -295,6 +291,8 @@ jobs:
   - ComplianceDebug
   condition: and(succeeded(), succeeded())
   pool: VSEng-MicroBuildVS2017
+  variables:
+    MicroBuild_NuPkgSigningEnabled: 'true'
```

Comment:
Why add signing here?

Reply:
oops--wrong phase

---
In reply to: [297877301](https://github.com/microsoft/axe-windows/pull/80#discussion_r297877301) [](ancestors = 297877301)